modules/metering-install-operator.adoc

```diff
@@ -5,21 +5,120 @@
 
 .Procedure
 
-. In the {product-title} web console,  click *Operators* -> *OperatorHub*, and filter for `metering` to find the
-Metering Operator.
+.  Create a namespace object YAML file for the Metering Operator with the `oc create -f <file-name>.yaml` command. You must use the CLI to create the namespace. For example, `metering-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-metering <1>
+  annotations:
+    openshift.io/node-selector: "" <2>
+  labels:
+    openshift.io/cluster-monitoring: "true"
+----
+<1> It is strongly recommended to deploy metering in the `openshift-metering` namespace.
+<2> Include this annotation before configuring specific node selectors for operand Pods.
 
-. Click the Metering card, review the package description, and then click *Install*.
-. Select an *Update Channel*, *Installation Mode*, and *Approval Strategy*.
-. Click *Subscribe*.
+.  In the {product-title} web console, click *Operators* -> *OperatorHub*. Filter for `metering` to find the Metering Operator.
 
-. On the *Installed Operators* screen, the *Status* displays *Succeeded* when the Metering Operator finishes installation. Click the name of the Operator in the first column to view the *Operator Details* page.
+.  Click the Metering card, review the package description, and then click *Install*.
+.  Select an *Update Channel*, *Installation Mode*, and *Approval Strategy*.
+.  Click *Subscribe*.
 
+.  Verify that the Metering Operator is installed by switching to the *Operators* -> *Installed Operators* page. The Metering Operator has a *Status* of *Succeeded* when the installation is complete.
++
 [NOTE]
 ====
-It might take a few minutes for the Metering Operator to appear.
+It might take several minutes for the Metering Operator to appear.
 ====
 
-From the *Operator Details* page you can create different resources related to metering. To complete your installation create a MeteringConfig resource to configure metering and install the components of the metering stack.
+. Click *Metering* on the *Installed Operators* page for Operator *Details*. From the *Details* page you can create different resources related to metering.
+
+To complete the metering installation, create a MeteringConfig resource to configure metering and install the components of the metering stack.
+
+[id="metering-install-cli_{context}"]
+== Installing metering using the CLI
+
+You can use the {product-title} CLI to install the Metering Operator.
+
+.Procedure
+
+. Create a namespace object YAML file for the Metering Operator. You must use the CLI to create the namespace. For example, `metering-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-metering <1>
+  annotations:
+    openshift.io/node-selector: "" <2>
+  labels:
+    openshift.io/cluster-monitoring: "true"
+----
+<1> It is strongly recommended to deploy metering in the `openshift-metering` namespace.
```

Comment:
Double checking - I thought the operator-metering namespace was required?

Reply:
> Double checking - I thought the operator-metering namespace was required?

<1> It is strongly recommended to deploy metering in the `openshift-metering` namespace.

@timflannagan1 can you please confirm?

Reply:
You could deploy Metering via OLM to any namespace and the operator would respect that, it's just a strong recommendation that you install/deploy the operator in the `openshift-metering` namespace.